packages/api/package.json

```diff
@@ -6,10 +6,14 @@
   "types": "./dist/index.d.ts",
   "private": true,
   "scripts": {
-    "build": "tsc -b"
+    "build": "tsc -b",
+    "generateSchema": "typescript-json-schema src/orbitAPI.ts ValidatableSpec -o src/orbitAPISchema.json --noExtraProps --required --ignoreErrors --strictNullChecks"
```

Comment:
`--ignoreErrors`: This is an unfortunate flag to use.

<details>
<summary> Without the flag this error is generated</summary>

```bash
orbit/node_modules/@types/react-native/globals.d.ts (49,13): Subsequent variable declarations must have the same type.  Variable 'Blob' must be of type '{ new (blobParts?: BlobPart[] | undefined, options?: BlobPropertyBag | undefined): Blob; prototype: Blob; }', but here has type '{ new (blobParts?: (string | Blob)[] | undefined, options?: BlobOptions | undefined): Blob; prototype: Blob; }'.
orbit/node_modules/@types/react-native/globals.d.ts (54,15): Duplicate identifier 'FormData'.
orbit/node_modules/@types/react-native/globals.d.ts (110,5): Subsequent property declarations must have the same type.  Property 'body' must be of type 'BodyInit | null | undefined', but here has type 'BodyInit_ | undefined'.
orbit/node_modules/@types/react-native/globals.d.ts (119,5): Subsequent property declarations must have the same type.  Property 'signal' must be of type 'AbortSignal | null | undefined', but here has type 'AbortSignal | undefined'.
orbit/node_modules/@types/react-native/globals.d.ts (137,14): Duplicate identifier 'RequestInfo'.
orbit/node_modules/@types/react-native/globals.d.ts (156,13): Subsequent variable declarations must have the same type.  Variable 'Response' must be of type '{ new (body?: BodyInit | null | undefined, init?: ResponseInit | undefined): Response; prototype: Response; error(): Response; redirect(url: string, status?: number | undefined): Response; }', but here has type '{ new (body?: BodyInit_ | undefined, init?: ResponseInit | undefined): Response; prototype: Response; error: () => Response; redirect: (url: string, status?: number | undefined) => Response; }'.
orbit/node_modules/@types/react-native/globals.d.ts (233,5): Subsequent property declarations must have the same type.  Property 'abort' must be of type 'ProgressEvent<XMLHttpRequestEventTarget>', but here has type 'ProgressEvent<EventTarget>'.
orbit/node_modules/@types/react-native/globals.d.ts (234,5): Subsequent property declarations must have the same type.  Property 'error' must be of type 'ProgressEvent<XMLHttpRequestEventTarget>', but here has type 'ProgressEvent<EventTarget>'.
orbit/node_modules/@types/react-native/globals.d.ts (235,5): Subsequent property declarations must have the same type.  Property 'load' must be of type 'ProgressEvent<XMLHttpRequestEventTarget>', but here has type 'ProgressEvent<EventTarget>'.
orbit/node_modules/@types/react-native/globals.d.ts (236,5): Subsequent property declarations must have the same type.  Property 'loadend' must be of type 'ProgressEvent<XMLHttpRequestEventTarget>', but here has type 'ProgressEvent<EventTarget>'.
orbit/node_modules/@types/react-native/globals.d.ts (237,5): Subsequent property declarations must have the same type.  Property 'loadstart' must be of type 'ProgressEvent<XMLHttpRequestEventTarget>', but here has type 'ProgressEvent<EventTarget>'.
orbit/node_modules/@types/react-native/globals.d.ts (238,5): Subsequent property declarations must have the same type.  Property 'progress' must be of type 'ProgressEvent<XMLHttpRequestEventTarget>', but here has type 'ProgressEvent<EventTarget>'.
orbit/node_modules/@types/react-native/globals.d.ts (239,5): Subsequent property declarations must have the same type.  Property 'timeout' must be of type 'ProgressEvent<XMLHttpRequestEventTarget>', but here has type 'ProgressEvent<EventTarget>'.
orbit/node_modules/@types/react-native/globals.d.ts (280,14): Duplicate identifier 'XMLHttpRequestResponseType'.
orbit/node_modules/@types/react-native/globals.d.ts (287,15): Duplicate identifier 'URL'.
orbit/node_modules/@types/react-native/globals.d.ts (312,15): Duplicate identifier 'URLSearchParams'.
orbit/node_modules/@types/react-native/globals.d.ts (346,5): Subsequent property declarations must have the same type.  Property 'onopen' must be of type '((this: WebSocket, ev: Event) => any) | null', but here has type '(() => void) | null'.
orbit/node_modules/@types/react-native/globals.d.ts (347,5): Subsequent property declarations must have the same type.  Property 'onmessage' must be of type '((this: WebSocket, ev: MessageEvent<any>) => any) | null', but here has type '((event: WebSocketMessageEvent) => void) | null'.
orbit/node_modules/@types/react-native/globals.d.ts (348,5): Subsequent property declarations must have the same type.  Property 'onerror' must be of type '((this: WebSocket, ev: Event) => any) | null', but here has type '((event: WebSocketErrorEvent) => void) | null'.
orbit/node_modules/@types/react-native/globals.d.ts (349,5): Subsequent property declarations must have the same type.  Property 'onclose' must be of type '((this: WebSocket, ev: CloseEvent) => any) | null', but here has type '((event: WebSocketCloseEvent) => void) | null'.
orbit/node_modules/@types/react-native/globals.d.ts (352,13): Subsequent variable declarations must have the same type.  Variable 'WebSocket' must be of type '{ new (url: string, protocols?: string | string[] | undefined): WebSocket; prototype: WebSocket; readonly CLOSED: number; readonly CLOSING: number; readonly CONNECTING: number; readonly OPEN: number; }', but here has type '{ new (uri: string, protocols?: string | string[] | null | undefined, options?: { [optionName: string]: any; headers: { [headerName: string]: string; }; } | null | undefined): WebSocket; prototype: WebSocket; readonly CLOSED: number; readonly CLOSING: number; readonly CONNECTING: number; readonly OPEN: number; }'.
orbit/node_modules/@types/react-native/globals.d.ts (376,15): Duplicate identifier 'AbortSignal'.
orbit/node_modules/@types/react-native/globals.d.ts (389,15): Duplicate identifier 'AbortController'.
orbit/node_modules/@types/react-native/globals.d.ts (414,14): Subsequent property declarations must have the same type.  Property 'error' must be of type 'DOMException | null', but here has type 'Error | null'.
orbit/node_modules/@types/react-native/globals.d.ts (422,14): Subsequent property declarations must have the same type.  Property 'result' must be of type 'string | ArrayBuffer | null', but here has type 'string | ArrayBuffer'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (2008,11): Duplicate identifier 'AbortController'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (2019,13): Duplicate identifier 'AbortController'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (2029,11): Duplicate identifier 'AbortSignal'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (2041,13): Duplicate identifier 'AbortSignal'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (5620,11): Duplicate identifier 'FormData'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (5630,13): Duplicate identifier 'FormData'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (16046,11): Duplicate identifier 'URL'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (16063,13): Duplicate identifier 'URL'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (16073,11): Duplicate identifier 'URLSearchParams'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (16106,13): Duplicate identifier 'URLSearchParams'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (20030,6): Duplicate identifier 'RequestInfo'.
orbit/node_modules/typescript/lib/lib.dom.d.ts (20230,6): Duplicate identifier 'XMLHttpRequestResponseType'.
(node:20182) UnhandledPromiseRejectionWarning: Error: No output definition. Probably caused by errors prior to this?
    at Object.<anonymous> (orbit/node_modules/typescript-json-schema/dist/typescript-json-schema.js:1290:23)
    at step (orbit/node_modules/typescript-json-schema/dist/typescript-json-schema.js:44:23)
    at Object.next (orbit/node_modules/typescript-json-schema/dist/typescript-json-schema.js:25:53)
    at orbit/node_modules/typescript-json-schema/dist/typescript-json-schema.js:19:71
    at new Promise (<anonymous>)
    at __awaiter (orbit/node_modules/typescript-json-schema/dist/typescript-json-schema.js:15:12)
    at Object.exec (orbit/node_modules/typescript-json-schema/dist/typescript-json-schema.js:1269:12)
    at Object.run (orbit/node_modules/typescript-json-schema/dist/typescript-json-schema-cli.js:53:30)
    at Object.<anonymous> (orbit/node_modules/typescript-json-schema/bin/typescript-json-schema:3:5)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:20182) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:20182) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
</details>

I did quite a bit of research to try and resolve these errors but the cause of them is unclear to me since we are not even importing/using most of the types described in the message. Regardless, the schema is generated correctly.